debian/control

```diff
@@ -3,7 +3,7 @@ Maintainer: Ralph Hempel <rhempel@hempeldesigngroup.com>
 Section: python
 Priority: optional
 Standards-Version: 3.9.5
-Build-Depends: python3-setuptools (>= 0.6b3), python3-all (>= 3.4), debhelper (>= 9), dh-python
+Build-Depends: python3-setuptools (>= 0.6b3), python3-all (>= 3.4), debhelper (>= 9), dh-python, python-evdev
```

Comment:
This is build deps. You don't need to add anything here.

Debian automatically figures out python package dependencies, so you don't need to add it in the binary package either.

(And this is the python2 version of the package)

Reply:
reverted